### PR TITLE
Branch 0.5

### PR DESCRIPTION
**Pull request by crazed7**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)
(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

Please review http://spark.apache.org/contributing.html before opening a pull request.

### CODE REVIEW

File: repl/src/main/scala/spark/repl/SparkILoop.scala [hunk 1]
```diff
@@ -200,7 +200,7 @@ class SparkILoop(in0: Option[BufferedReader], val out: PrintWriter, val master:
       ____              __  
      / __/__  ___ _____/ /__
     _\ \/ _ \/ _ `/ __/  '_/
-   /___/ .__/\_,_/_/ /_/\_\   version 0.5.2-SNAPSHOT
```
Comment (khanm002) [inline repl/src/main/scala/spark/repl/SparkILoop.scala]: #-   /___/ .__/\_,_/_/ /_/\_\   version 0.5.2-SNAPSHOT

**Comment by srowen** [comment]
@khanm002 close this

**Review by crazed7 — COMMENTED** [review]
Making the necessary changes

**Comment by HyukjinKwon** [comment]
@khanm002, could you close this please?

If you are going to propose a change, please follow this linke - https://cwiki.apache.org/confluence/display/SPARK/Contributing+to+Spark.

**Comment by vanzin** [comment]
@khanm002 please close this.

**Pull request closed without merging**